From e25de2de74be09071a8f752d38263cab9c316254 Mon Sep 17 00:00:00 2001
From: Peter Stephenson <p.stephenson@samsung.com>
Date: Thu, 14 Feb 2019 11:41:19 +0000
Subject: [PATCH] c.f. 44062: Back off clearflag change to ZLE line init.

This causes problems with failed ZLE reads that printed a message
in the display area, in particular ignoreEOF warnings.

It appears it's not needed for the overall effect of the fixes in
zsh-workers/40302, commit 34656ec2.
---
 Src/Zle/zle_main.c | 1 -
 2 files changed, 6 insertions(+), 1 deletion(-)

diff --git a/Src/Zle/zle_main.c b/Src/Zle/zle_main.c
index 3487b5d9f..71930f76b 100644
--- a/Src/Zle/zle_main.c
+++ b/Src/Zle/zle_main.c
@@ -1256,7 +1256,6 @@ zleread(char **lp, char **rp, int flags, int context, char *init, char *finish)
     resetneeded = 0;
     fetchttyinfo = 0;
     trashedzle = 0;
-    clearflag = 0;
     raw_lp = lp;
     lpromptbuf = promptexpand(lp ? *lp : NULL, 1, NULL, NULL, &pmpt_attr);
     raw_rp = rp;
-- 
2.21.0-rc2

